cmd/rpcdaemon/commands/send_transaction.go

```diff
@@ -36,8 +37,9 @@ func (api *APIImpl) SendRawTransaction(ctx context.Context, encodedTx hexutility
 	}
 	chainId := cc.ChainID
 
-	// [zkevm] - proxy the request if the chainID is ZK
-	if zkchainconfig.IsZk(chainId.Uint64()) {
+	// [zkevm] - proxy the request if the chainID is ZK and not a sequencer
```

Comment:
I wonder if we can extract this to `send_transaction_zkevm.go`?

Reply:
I have moved the bulk of this over to another file, there is still a small if statement in the call however to avoid an `eth_sendRawTransactionZk` call